project/plugins.sbt

```diff
@@ -2,26 +2,26 @@ resolvers += Resolver.url("HMRC Sbt Plugin Releases", url("https://dl.bintray.co
 
 resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += "Typesafe Releases" at "httpssb://repo.typesafe.com/typesafe/releases/"
```

Comment:
should be http

Reply:
Fixed. Actually https as http is now deprecated